changelog/README.md

```diff
@@ -0,0 +1,81 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+*This strategy works for the libnavui-androidauto project too. It works in the `libnavui-androidauto/changelog` directory*
+
+To follow this strategy you should create a `.md` file for every PR. Choose a directory:
+
+- `changelog/unreleased/features` for **Features** changes
```

Comment:
IMO would be nice to have a script to add changelog like `python add_changelog.py -f "Changelog string"`, where `-f` is `feature` (also should be other flags), `"Changelog string"` is the entry of changelog

Reply:
done